loc/translations-import/vscode-powerplatform.zh-CN.xlf

```diff
@@ -256,41 +243,14 @@ The second line should be '[TRANSLATION HERE](command:pacCLI.pacAuthHelp)', keep
         <note>The {0} represents auth profile's user name (email address))</note>
         <target state="translated">用户: {0}</target>
       </trans-unit>
-      <trans-unit id="pacCLI.SolutionTreeItem.label">
-        <source xml:lang="en">{0}, Version: {1}</source>
-        <note>The {0} represents Solution's Friendly / Display name
-The {1} represents Solution's Version number</note>
-        <target state="translated">{0}，版本: {1}</target>
-      </trans-unit>
-      <trans-unit id="pacCLI.SolutionTreeItem.toolTip">
+      <trans-unit id="pacCLI.EnvOrSolutionTreeItem.toolTip">
```

Comment:
Missing translations for `pacCLI.EnvOrSolutionTreeItem.toolTip` across all languages.

Reply:
Ignoring for now, Translation team will get the string in another PR